internal/database/migrations/20230113-0000.go

```diff
@@ -0,0 +1,97 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"github.com/google/uuid"
+	"github.com/lib/pq"
+	"time"
+)
+
+func addApexTables() *gormigrate.Migration {
+
+	// Migrations rules:
+	//
+	//  1. IDs are numerical timestamps that must sort ascending.
+	//     Use YYYYMMDD-HHMM w/ 24 hour time for format
+	//     Example: August 21 2018 at 2:54pm would be 20180821-1454.
+	//
+	//  2. Include models inline with migrations to see the evolution of the object over time.
```

Comment:
That seems contrary to what `gomigrate` says, where when adding a column we include a partial struct:
```go
                {
			ID: "201608301415",
			Migrate: func(tx *gorm.DB) error {
				// when table already exists, it just adds fields as columns
				type Person struct {
					Age int
				}
				return tx.AutoMigrate(&Person{})
			},
			Rollback: func(tx *gorm.DB) error {
				return tx.Migrator().DropColumn("people", "age")
			},
		},
```